### PR TITLE
Added editing events to CSteamGroup

### PR DESCRIPTION
**Pull request by luop90**
This adds the ability to edit events from a given id. Takes the exact same parameters as createEvent(), plus an id.

### CODE REVIEW

File: components/groups.js [hunk 1]
```diff
@@ -286,6 +286,64 @@ SteamCommunity.prototype.scheduleGroupEvent = function(gid, name, type, descript
...
+	// Event types: ChatEvent - Chat, OtherEvent - A lil somethin somethin, PartyEvent - Party!, MeetingEvent - Important meeting, SpecialCauseEvent - Special cause (charity ball?), MusicAndArtsEvent - Music or Art type thing, SportsEvent - Sporting endeavor, TripEvent - Out of town excursion
+	// Passing a number for type will make it a game event for that appid
+
+	if (typeof server === 'function') {
+		callback = server;
+		server = {"ip": "", "password": ""};
+	} else if (typeof server === 'string') {
+		server = {"ip": server, "password": ""};
+	} else {
+		server = {"ip": "", "password": ""};
```
Comment (DoctorMcKay) [inline components/groups.js]: I realize this is copied from `scheduleGroupEvent` (where it also appears to be incorrect), but doesn't this effectively break the `{"ip": "123.123.123.123", "password": "foobar"}` usage for `server`?

**Comment by luop90** [comment]
Yes, there isn't a check to see if the server is already a valid object. However, that part was copied from `scheduleGroupEvent `(as you said), so should I fix it in both places for this PR?

**Comment by DoctorMcKay** [comment]
>should I fix it in both places for this PR

If you wouldn't mind, it would be greatly appreciated.

**Comment by luop90** [comment]
Done!

**Comment by DoctorMcKay** [comment]
You even kept the style the same. Thanks! :+1:

**Pull request merged**